### PR TITLE
Improve documentation on how to start locally

### PR DESCRIPTION
**Pull request by armenzg**
Mention that grunt to will serve and show your changes live.
Mention that npm install is needed to install all packages.

### CODE REVIEW

**Comment by armenzg** [comment]
@jonasfj r?

File: README.md [hunk 1]
```diff
@@ -27,15 +26,24 @@ Our grunt file offers compilation of:
...
 
-targets:
+Available targets
+-----------------
 
   - `grunt`, the default target build, watches sources and serves the `build/`
     folder on `http://localhost:9000/`.
   - `grunt build`, builds sources into the `build/` folder.
   - `grunt clean`, delete generated files (ie. deletes the `build/` folder).
+  - `grunt develop`, it serves on localhost and watches sources
```
Comment (gregarndt) [inline README.md]: so what I'm not clear on is how `grunt` and `grunt develop` differ...seems that both serves up on localhost and watches sources

Comment (jonasfj) [inline README.md]: They are the same :)
https://github.com/taskcluster/taskcluster-tools/blob/948e474354c7b066a18591c8ba164f893c17e744/Gruntfile.js#L219

**Comment by jonasfj** [comment]
Awesome, thanks :)

**Pull request merged**